Sources/StevensonCore/GHRepo.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import Vapor
+
+typealias Version = String
+
+public struct GHRepo {
+    public let key: String
```

Comment:
what does `key` stands for? is it the access token?

Reply:
It's the keyword that will identify the repo uniquely especially to pass it as an argument in Slack when invoking the slash command (`/crp <repoKey> <branch>`)

What do you think about `identifier` instead? Or is `name` better?

Reply:
it seems to me that the name of the repo is the most obvious to use for such key, so I'd name it the same way. or it can/should be a different value?